nautobot/extras/tests/test_jobs.py

```diff
@@ -348,6 +348,18 @@ def test_job_data_as_string(self):
         ).first()
         self.assertIn("Data should be a dictionary", log_failure.message)
 
+    def test_job_latest_result_property(self):
+        """
+        Job test to see if the latest_result property is indeed returning the most recent job result
+        """
+        module = "test_pass"
+        name = "TestPass"
+        job_result = create_job_result_and_run_job(module, name, commit=False)
+        self.assertEqual(job_result.status, JobResultStatusChoices.STATUS_COMPLETED)
+        job_class, job_model = get_job_class_and_model(module, name)
+        latest_job_result = job_model.results.first()
```

Comment:
Should this be `job_model.latest_result`? Also, should we ensure that there are more than one JobResult present for this job so as to actually test the `latest_result` property?

Reply:
Im not too sure if we are testing the `latest_result()` property or making sure that the` first()` function is getting the most recent job result as opposed to `last()` before. But i think` job_model.latest_result `should def be more comprehensive. And yeah sorry for the oversight we def should make sure there are more than job results. Thank you!